src/owr/ObolErc1155Recipient.sol

```diff
@@ -282,7 +235,7 @@ contract ObolErc1155Recipient is ERC1155, Ownable, IERC1155Receiver {
     // claim before transfer
     uint256 length = ids.length;
     for (uint256 i; i < length; i++) {
-      _claim(ids[i]); //allow transfer even if `claimed == 0`
+      ownerOf[ids[i]] = to;
```

Comment:
We don't need this. it can be replaced with `balanceOf()` check directly

Reply:
this doesn't seem to exist in the latest version